### PR TITLE
[NEW] Create Rust blinky lights workshop

### PR DESCRIPTION
**Pull request by davidbeechey**
Workshop intended to be run _after_ the "Getting Started with Rust" workshop as an introduction to writing embedded Rust.

### CODE REVIEW

File: 03-rust_blinky_lights/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  let p = embassy_stm32::init(Default::default());
+```
+
+### The LED
+
+```rust
+    let mut led = Output::new(p.PA5, Level::High, Speed::Low);
+```
+
+This creates a new `Ouput` struct, which represents a GPIO pin that can be written to. The first argument is the pin, the second is the initial state of the pin, and the third is the speed of the pin.
```
Comment (kshxtij) [inline 03-rust_blinky_lights/README.md:45]: Output

**Review by kshxtij — CHANGES_REQUESTED** [review]
LGTM, i spotted a type but just run a spell check just in case tbh

**Review by TomLonergan03 — APPROVED** [review]
very nice

**Review by licornes-fluos — APPROVED** [review]
Seems great

**Pull request merged**